src/Shared/Components/SelectPicker/FilterSelectPicker.tsx

```diff
@@ -85,6 +88,8 @@ const FilterSelectPicker = ({
                 isClearable={false}
                 customSelectedOptionsCount={appliedFiltersCount}
                 icon={filterIcon}
+                inputValue={inputValue}
+                onInputChange={setInputValue}
```

Comment:
The onInputChange for multiple options like set option, create option etc. Do we need this for all of them?